pkg/enhancers/github/github.go

```diff
@@ -0,0 +1,41 @@
+package github
+
+import (
+	"github.com/argonsecurity/pipeline-parser/pkg/enhancers"
+	"github.com/argonsecurity/pipeline-parser/pkg/models"
+	"github.com/pkg/errors"
+)
+
+type GitHubEnhancer struct{}
+
+func (g *GitHubEnhancer) LoadImportedPipelines(data *models.Pipeline, credentials *models.Credentials) ([]*enhancers.ImportedPipeline, error) {
+	var errs error
+	importedPipelines, err := getReusableWorkflows(data, credentials)
+	if err != nil {
+		errs = errors.Wrap(errs, err.Error())
+	}
+
+	return importedPipelines, errs
+}
+
+func (g *GitHubEnhancer) Enhance(data *models.Pipeline, importedPipelines []*enhancers.ImportedPipeline, credentials *models.Credentials) (*models.Pipeline, error) {
```

Comment:
`credentials *models.Credentials` it seems you don't use this param in this function